src/model/views/reservation_view.py

```diff
@@ -109,8 +106,8 @@ def update_reservation(reservation_id: int,
                 reservation_status_id = {reservation_status_id},
                 room_number_of_adults = {reservation_number_of_adults},
                 room_number_of_children = {reservation_number_of_children},
-                reservation_start_date = '{reservation_start_date.strftime('%d-%m-%Y 15:00:00')}',
-                reservation_end_date = '{reservation_end_date.strftime('%d-%m-%Y 12:00:00')}',
+                reservation_start_date = '{reservation_start_date.strftime('%Y-%m-%d 15:00:00')}',

```

Comment:
Możliwe, że ta zmiana spowoduje błędy po stronie UI. Po stronie UI ja przesyłam datę w formacie 'dd-MM-yyyy'. Jak chcesz zostaw. Najwyżej po mergu ja poprawię na front'cie

Reply:
Niee, ta funkcja wyciąga tylko date w konkretnym formacie z obiektu datetime, na frontend to nie ma wpływu.